recipe/meta.yaml

```diff
@@ -11,10 +11,12 @@ source:
     folder: rmm
 
 build:
-  number: 1
+  number: 2
   skip: true  # [(not linux) or (cuda_compiler_version == "None")]
   run_exports:
     - {{ pin_subpackage(name, max_pin="x.x") }}
+    - {{ pin_subpackage("thrust", exact=True) }}
+    - {{ pin_subpackage("cub", exact=True) }}
```

Comment:
Do Thrust and Cub do patch releases that guarantee API/ABI stability that we'd want to allow to be picked up?

Reply:
This was the question I wanna ask. It seems we must use `pin_compatible` instead of `pin_subpackage`.

Thrust/CUB does occasionally have patch releases in the past, but not recently. We definitely want to pin `<2`.